env.ini

```diff
@@ -3,4 +3,4 @@
 # open a user account on https://cs.degirum.com. Please login to your 
 # account and go to the token generation page to generate API access token.
 #
-# DEGIRUM_CLOUD_TOKEN = "paste your cloud API token here"
+DEGIRUM_CLOUD_TOKEN = "dg_8reeVK2owBM6TRr2tdXc9YfRbbb1cGV4qjTMM"
```

Comment:
Better to remove your token from here :)

Reply:
@vlad-nn this is why I do not like env.ini approach. There is always a chance that this will be checked in especially when we press stage all changes. If we can find some approach like user secret or os.environment setup for token, it would eliminate this problem.